liquibase-standard/src/main/java/liquibase/change/AbstractSQLChange.java

```diff
@@ -351,4 +357,117 @@ public void close() throws IOException {
             stream.close();
         }
     }
+
```

Comment:
Hm... would not be better to have this NormalizingStreamV8 class in a separated file? It's so big .